lib/td/command/export.rb

```diff
@@ -103,6 +103,8 @@ def table_export(op)
       raise ArgumentError, "#{s} is not a supported encryption method" unless SUPPORTED_ENCRYPT_METHOD.include?(s)
       encryption = s
     }
+    # Increase the summary_width value from default 32 into 36, to show the options help message properly.
+    op.summary_width = 36
```

Comment:
IMO it's better if this line is put below this line https://github.com/treasure-data/td/blob/8354f3f694953b0956b10297ef7d78803561205d/lib/td/command/runner.rb#L25

Reply:
@tung-vu-td Thanks for your advice!

I moved the configuration into the place you suggested.
By the way, if we move to there, then the configuration will be applied into all commands, not only `export` command.
The impact will be getting much much bigger. Little bit worry about it.

Could you please do double check?

Additionally the CircleCI job is failed again, please take a look. Thanks!

Reply:
hmmm...

It looks like don't work if we move the configuration into runner.rb file.
Let's try to investigate more in details...

Reply:
After merge the master branch, the all checks have passed in CircleCI. Thanks!

Reply:
@kwangshin I agree that not all command need to increase summary width. Let's put the line before all the op.on functions, aka before this line https://github.com/treasure-data/td/blob/c55b8dcb3bfeb3c373a7c273ef1bb253723e0b10/lib/td/command/export.rb#L70

Reply:
@tung-vu-td Finally we are in the same line! 😄 

I fully agree with your suggestion. As it's the configuration of op, it's good idea to add it before all the op.on functions. 👍 

Please review and merge when you're free. Cheers!